app/grandchallenge/evaluation/tasks.py

```diff
@@ -271,6 +272,13 @@ def create_algorithm_jobs_for_evaluation(*, evaluation_pk, max_jobs=1):
         requires_memory_gb=evaluation.submission.algorithm_requires_memory_gb,
     )
 
+    job_utilizations = JobUtilization.objects.filter(job__in=jobs)
+    job_utilizations.update(
+        phase=evaluation.submission.phase,
+        archive=evaluation.submission.phase.archive,
+        challenge=evaluation.submission.phase.challenge,
+    )
```

Comment:
`create_algorithm_jobs` can raise `TooManyJobsScheduled`, in which case this metadata will not get added.